### PR TITLE
Overhauled README to be more accurate/easy to read

### PR DESCRIPTION
**Pull request by xomiamoore**

### CODE REVIEW

**Comment by akeller** [comment]
This may just be because I haven't looked into org level READMEs, but why is this in the profile directory? Is this just how this works?

**Comment by xomiamoore** [comment]
Yeah, it's different for orgs, for whatever reason:

```
camunda-community-hub/.github is a special repository.

The /profile/README.md will appear on the organization's profile.
```

File: profile/README.md [hunk 1]
```diff
@@ -1,48 +1,40 @@
-# The Camunda Community Hub
+# Welcome to the Camunda Community Hub!
+## A project to empower our open source community extension maintainers, and encourage new contributors to get started contributing to the Camunda open source ecosystem.
```
Comment (akeller) [inline profile/README.md]:
```suggestion
A project to empower our open source community extension maintainers and encourage new contributors to get started contributing to the Camunda open source ecosystem. 
```

This is a bit of a nitpick/suggestion, but also important for accessibility - Only add headers to headers.

File: profile/README.md [hunk 2]
```diff
@@ -1,48 +1,40 @@
-# The Camunda Community Hub
+# Welcome to the Camunda Community Hub!
+## A project to empower our open source community extension maintainers, and encourage new contributors to get started contributing to the Camunda open source ecosystem. 
 
-The Camunda Community Hub is Camunda's GitHub Organization for community contributed extensions. Those extensions are independently maintained by members of the Camunda open source community (with some maintainers happen to be Camunda employees too).
+Extensions in the Camunda Community Hub organization are independently led and maintained projects by members of the broader Camunda open source community, some of whom happen to be Camunda employees, or are projects/tutorials that are created and maintained by the Camunda Developer Relations team!
```
Comment (akeller) [inline profile/README.md:4]: I love this call out! 🧡

File: profile/README.md [hunk 3]
```diff
@@ -1,48 +1,40 @@
...
 
-With the Camunda Community Hub we want to empower our open source community to contribute, and encourage new contributors to get started contributing to the Camunda open source ecosystem.
+With the Camunda Community Hub, we want to empower our open source community to contribute, and encourage new contributors to get started contributing to the Camunda open source ecosystem.
 
-## Existing community extensions
-
-You can find a list of existing community extensions by [browsing the repostories](https://github.com/orgs/camunda-community-hub/repositories) in this GitHub organization.
+**Table of Contents**
```
Comment (akeller) [inline profile/README.md]: This could be a header instead of bold.

File: profile/README.md [hunk 4]
```diff
@@ -1,48 +1,40 @@
...
+## Existing community extensions
 
-## How to get my community extension to the Camunda Community Hub?
+You can find a list of existing community extensions by [browsing the repostories](https://github.com/orgs/camunda-community-hub/repositories) in this GitHub organization.
 
-- [Open an issue](https://github.com/Camunda-Community-Hub/community/issues/new/) in the [community repository](https://github.com/Camunda-Community-Hub/community/) and ask to be invited to join the Camunda Community Hub (if you are not already a collaborator).
+You can also browse by Awesome Lists created by the community.
+- [Awesome Camunda External Task Clients](https://github.com/camunda-community-hub/awesome-camunda-external-clients)
+- [Awesome Camunda Platform 8 Projects](https://github.com/camunda-community-hub/awesome-camunda-platform-8)
+- [Awesome Camunda and Robot Framework Projects](https://github.com/camunda-community-hub/awesome-Camunda-and-Robotframework-projects)
```
Comment (akeller) [inline profile/README.md]: You may want to make it a little clearer about the 2 Awesome lists that are for C7. We made some choices during the C8 launch to be pretty overt about this - explicitly name things for C7 and C8. This is a recommendation, but you could easily add something to describe each of these Awesome lists directly in the list.

File: profile/README.md [hunk 5]
```diff
@@ -1,48 +1,40 @@
...
 
-- Sign our [CLA](https://cla-assistant.io/camunda-community-hub/community) and agree to our [Code of Conduct](https://camunda.com/events/code-conduct/). 
+## How to contribute
+- [Open an issue](https://github.com/Camunda-Community-Hub/community/issues/new/) in the [community repository](https://github.com/Camunda-Community-Hub/community/) and ask to be invited to join the Camunda Community Hub (if you are not already a member).
+- Read our [Contributor's Guide](https://github.com/camunda-community-hub/community/blob/a6526a0f07893aa84434a2cadd9ffdc332a1f89d/CONTRIBUTING.MD#L25).
```
Comment (akeller) [inline profile/README.md]: This link looks like it includes a commit or blob URL. I would use this one instead - `https://github.com/camunda-community-hub/community/blob/main/CONTRIBUTING.MD`.

**Review by akeller — CHANGES_REQUESTED** [review]
Left a few comments, nitpicks, and suggestions! This is looking gooooood 🚀

**Review by akeller — APPROVED** [review]

**Comment by akeller** [comment]
Since you just logged off for the weekend and I'm still here, Imma merge it 🚀

**Comment by xomiamoore** [comment]
@allcontributors add @akeller for review please!

**Pull request merged**